codespell_lib/_codespell.py

```diff
@@ -1028,7 +1029,10 @@ def main(*args: str) -> int:
     else:
         ignore_word_regex = None
 
-    ignore_words_files = options.ignore_words or []
+    ignore_words_files = []
+    if options.ignore_words:
+        for iw in options.ignore_words:
+            ignore_words_files.extend(iw.split(","))
```

Comment:
Remove a branch...
```suggestion
    for iw in options.ignore_words to []:
        ignore_words_files.extend(iw.split(","))
```

Reply:
You fold the branch by changing `if` to `or`. Complexity is not really reduced.

I believe this a shortcoming of the complexity metrics, and this looks like a trick that makes use of that shortcoming, instead of tackling real complexity.